.travis.yml

```diff
@@ -13,7 +13,8 @@ script: rake ci
 # environment
 language: ruby
 rvm:
-  - "2.2"
+  - "2.2.3"
```

Comment:
@eefahy this is important--Jekyll has a dependency on `ActiveSomethingOrOther` that only works with Ruby >2.2.3. I think that's what's on the server, but FYI.